### ISSUE TITLE
Unable to delete the numbers entirely in Number Input Field

### ISSUE THREAD

**Issue by abhijit-kar** [pablo-abc/felte#15]
![Screenshot from 2021-04-01 09-59-29](https://user-images.githubusercontent.com/25662120/113243634-fff48580-92d0-11eb-8e5d-58685928afc5.png)

```html
<input
  type="number"
  name="min"
  bind:value="{$data.salary.min}"
  placeholder="0.00" />
```

**Comment by pablo-abc** [comment]
This is definitely a bug on my side. Seems to be interpreting an empty field as `0`. That's something I'm already fixed.

On another note, `bind:value` shouldn't be necessary anymore in this case. Is It still giving you issues?

Finally, I do believe that an `input[type=number]` is only recommended for values that increase/decrease in steps (which is why browsers add the arrows up and down). In most cases I've seen people recommend using an `<input type="text" inputmode="numeric">`. Here's a [great explanation of why](https://technology.blog.gov.uk/2020/02/24/why-the-gov-uk-design-system-team-changed-the-input-type-for-numbers/) if you're interested.

**Comment by abhijit-kar** [comment]
Thanks @pablo-abc,

I got to learn something new. Also, inputmode doesn't have all browser support, but still I used it everywhere its applicable.

I settled with textfield instead of a number field and let Yup do the validation with number type.

**Comment by pablo-abc** [comment]
I'm reopening since there's still the issue regarding to the number field being parsed as `0`.

**Comment by pablo-abc** [comment]
Fixed now in felte@0.6.3, an empty field or an invalid number should get set as `undefined` now.
